### PR TITLE
Update LICENSE date to new year

### PR DESCRIPTION
**Pull request by vasilvestre**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 Kévin Dunglas
+Copyright (c) 2019 Kévin Dunglas
```
Comment (dunglas) [inline LICENSE]: We can use the following, same for other PRs:
```suggestion
Copyright (c) 2018-present Kévin Dunglas
```

Comment (vasilvestre) [reply LICENSE]: Being copyright I think there's law thing to look at

Comment (vasilvestre) [reply LICENSE]: https://www.copyrightlaws.com/copyright-symbol-notice-year/
It should be a range anyway, sorry for the fast PR.
I've searched in github, it seem like present isn't common.
As you want so

Comment (dunglas) [reply LICENSE]: https://github.com/facebook/react/commit/b87aabdfe1b7461e7331abb3601d9e6bb27544bc

We should drop the year as Facebook do (anyway US copyright laws are not applicable in the EU, were this project is based).

Comment (vasilvestre) [reply LICENSE]: Seem totally fine

Comment (vasilvestre) [reply LICENSE]: I've modified it according to it. Could you resolve conversation if you are ok with it?

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**